docs/register-intune-device-id.md

```diff
@@ -10,12 +10,12 @@ Users with a non-SE GSIB device and onboarding their Internet Device to SEED.
 
 ## Prerequisites
 
-Following are the prerequisites to register Intune device ID on TechPass portal:
+You need the following to register your Intune Device ID on the TechPass portal:
 
-- An active TechPass account.
-- A non-SE GSIB device.
-- Complete setting up of Microsoft Intune on the Internet Device.
-- Intune Device ID of the Internet Device.
+- Active TechPass account.
+- Non-SE GSIB device.
+- Ensure you have set up Microsoft Intune on the Internet Device (device you are onboarding to SEED).
```

Comment:
is this point necessary? the next pt subsumes this point?

Reply:
fixed